bin/photos.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+'use strict'
+
+var easyimage = require('easyimage')
+var fs = require('fs')
+
+fs.readdir('./images/gallery', function (err, items) {
```

Comment:
looks like this is still going over the images gallery folder. 
Can this just look at the json file for any url without a thumbnail and update the json?

That way we can pull the urls into the json file and automate the whole process.